substrate/client/tracing/src/logging/mod.rs

```diff
@@ -182,12 +182,14 @@ where
 	let enable_color = force_colors.unwrap_or_else(|| io::stderr().is_terminal());
 	let timer = fast_local_time::FastLocalTime { with_fractional: detailed_output };
 
+	console::set_colors_enabled(enable_color);
```

Comment:
```suggestion
	// We need to set both together, because we are may printing to `stdout` and `stderr`.
	console::set_colors_enabled(enable_color);
```